ReflexMPG/Input_GameCube.h

```diff
@@ -240,10 +240,10 @@ class ReflexInputGameCube : public RZInputModule {
               | (gcdata[i].report.y ? GAMEPAD_MASK_B3 : 0) // Generic: P1, Switch: Y, Xbox: X
               | (gcdata[i].report.x ? GAMEPAD_MASK_B4 : 0) // Generic: P2, Switch: X, Xbox: Y
               //| (L      ? GAMEPAD_MASK_L1 : 0) // Generic: P4, Switch: L, Xbox: LB
-              //| (gcdata[i].report.z      ? GAMEPAD_MASK_R1 : 0) // Generic: P3, Switch: R, Xbox: RB
+              | (gcdata[i].report.z      ? GAMEPAD_MASK_R1 : 0) // Generic: P3, Switch: R, Xbox: RB

```

Comment:
The line of code is identical. Only the comment changes. Did you build and test the change?